### PR TITLE
patch envelopes: add test case covering deleting all PE's

### PR DESCRIPTION
**Pull request by uncleDecart**
Corner case when user de-attaches all the patch envelopes from app instance, there should be no PEs in description.json endpoint

### CODE REVIEW

File: pkg/pillar/cmd/msrv/patchenvelopes_test.go [hunk 1]
```diff
@@ -238,4 +238,37 @@ func TestPatchEnvelopes(t *testing.T) {
...
+
+			if len(envelopes2) == 0 && len(envelopes1) == 0 {
+				close(finishDeleting)
+				return
+			}
+			time.Sleep(time.Second)
+		}
+	}()
+
+	g.Eventually(func() bool {
```
Comment (milan-zededa) [inline pkg/pillar/cmd/msrv/patchenvelopes_test.go:263]: Btw. you can avoid the Go routine and simplify this to:
```
g.Eventually(func() bool {
    envelopes1 := peStore.Get(app1UUID).Envelopes
    envelopes2 := peStore.Get(app2UUID).Envelopes
    return len(envelopes2) == 0 && len(envelopes1) == 0
}, deadline, time.Second).Should(gomega.BeTrue())
```

**Review by OhmSpectator — APPROVED** [review]
